tasks/beats-config.yml

```diff
@@ -85,3 +85,15 @@
     group: root
   when: default_ilm_policy is defined
   notify: restart the service
+
+- name: Assert modules.path presence
+  assert:
+    that:
+      - beat_conf.filebeat.config.modules.path is defined
+    fail_msg: 'You need to configure beat_conf.filebeat.config.modules.path for modules to be loaded'
```

Comment:
same as above

Reply:
https://github.com/stoffus/ansible-beats/pull/1

Reply:
Thanks, good catch.

Reply:
Sorry, failed to sign-off the merge commit. Created a new one.